README.md

```diff
@@ -130,7 +134,19 @@ Where:
 
 ## Profile Retrieval
 
-When your application receives a token via the exposed endpoint (it will be assigned to a query string parameter named `token`), you can easily retrieve the user profile by adding the following to your endpoint handler:
+### Callback URl
+
+To allow sharing a Users profile with your application, you must expose a GET endpoint over https that Yoti will use to push a one-time token to your back end.
+
+You can use any address, path or port in your URL, but it must be https and it must allow a query string parameter named `token` - that's the value you wil need for retrieving the shared profile.
```

Comment:
We need to rephrase this:
`You can use any public domain name or localhost, as well as any path or port, but the resulting url must be served over https. Your endpoint must consume a query string parameter named 'token' - that's the value you will need for retrieving the shared profile`

Reply:
Done.